src/services/__tests__/userServices.test.js

```diff
@@ -19,4 +20,27 @@ describe('UserServices', () => {
       done();
     });
   });
+
+  it('should return default locale if no locale cookie is present', done => {
+    userServices.getLocale().then(locale => {
+      expect(locale).toMatchSnapshot();
+      done();
+    });
+  });
+
+  it('should return en locale if cookie is set to en_US', done => {
```

Comment:
optional: maybe make a more generic description... `should return a specific locale cookie value`